web/src/pages/onboarding.tsx

```diff
@@ -257,10 +258,10 @@ const OnboardingPage = (props: Props): ReactElement => {
       analytics.event.onboarding_last_step.submit.finish_onboarding();
     }
 
-    const isRemoteSellerWorker =
-      newProfileData.businessPersona === "FOREIGN" &&
-      (determineForeignBusinessType(newProfileData.foreignBusinessTypeIds) === "REMOTE_SELLER" ||
-        determineForeignBusinessType(newProfileData.foreignBusinessTypeIds) === "REMOTE_WORKER");
+    const isRemoteSellerWorker = isRemoteWorkerOrSellerBusiness({
+      ...updateQueue.currentBusiness(),
```

Comment:
This is a dope solution. You could also use `determineForeignBusinessType`. However, I have some thoughts on the function. See below.

In the ticket, I made a note to consider where the parameter for the `businessPersonalHelpers function` should be the ProfileData object. I noted this because we are using `determineForeignBusinessType` in the app and I was wondering if it's possible to kill that function and use one of the ones in the `businessPersonalHelpers` file. In areas like the profile data text field, we only have access to profile data.

What do you think? If we keep it, I think `determineForeignBusinessType` function should also be move to the `businessPersonalHelpers` file. It would make it easier to find.